### PR TITLE
Added exchange-transaction-receipts.md

### PR DESCRIPTION
**Pull request by ansnunez**

### CODE REVIEW

File: docs/exchange-transaction-receipts.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+"receipt": {
+  "cumulative_gas": "10481",
+  "epoch_num": "586524",
+  "success": true
+},
+```
+
+## Additional Fields
+
+For contract transactions, more information is contained in additional fields under the "**receipt**".
```
Comment (bb111189) [inline docs/exchange-transaction-receipts.md]:
```suggestion
For smart contract transactions, additional information relating to smart contract execution is contained in additional fields under the "**receipt**".
```

File: docs/exchange-transaction-receipts.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+### Successful Transactions
+
+If a transaction is successful (i.e., the **success** field is `true`), these fields will be present:
+
+| Field              | Type       | Description                                                                                  |
+|:------------------ |:---------- |:-------------------------------------------------------------------------------------------- |
+| **accepted**       | boolean    | Indicates whether the last transition in this transaction incurred a balance transfer        |
```
Comment (bb111189) [inline docs/exchange-transaction-receipts.md]:
```suggestion
| **accepted**       | boolean    | Indicates whether the last transition in this transaction incurred a **balance transfer**        |
```

File: docs/exchange-transaction-receipts.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+      }
+    }
+  ]
+}
+
+```
+
+### Unsuccessful Transactions
+
+If a transaction is unsuccessful (i.e., the **success** field is `false`), these fields will be present:
```
Comment (bb111189) [inline docs/exchange-transaction-receipts.md]:
```suggestion
If a transaction is unsuccessful (i.e., the **success** field is `false`) and no balance transfer will be executed.

These fields will be present:
```

File: docs/exchange-transaction-receipts.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+
+## Multiple Transitions with Balance Transfers
+
+For a transaction that involves multiple transitions that also result in balance transfers, these are the recommended steps to perform when reading the **receipt**:
+
+1. Confirm that the **success** field is set to `true`
+1. Traverse the **transitions** JSON array. For each transition:
+   1. Note the **addr** and the **_tag** called under **msg**
+   1. Find the contract referred to by the **addr**
+   1. Check whether the **_tag** in the contract is a transition that performs a balance transfer
```
Comment (bb111189) [inline docs/exchange-transaction-receipts.md]: If it doesn't not allow balance transfer, success is false right?

File: docs/exchange-transaction-receipts.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+    {
+      "line": 137,
+      "message": "Raised from ConfigureUsers"
+    }
+  ],
+  "success": false
+  }
+```
+
+## Multiple Transitions with Balance Transfers
```
Comment (bb111189) [inline docs/exchange-transaction-receipts.md]: Another new section
## Recommendation for exchanges polling for incoming deposit from smart contract transactions
1. Confirm that the **success** field is set to `true`
1. Traverse the **transitions** JSON array. For each transition, for a successful deposit of `$ZIL` via the smart contract, the following must be fulfilled:
   1. If the `_recipient` corresponds to a known exchange deposit address you controlled
   1. ` _tag` is either `AddFunds` or empty
   1. `_amount` is non zero
In such a case, you can confirm that there is a deposit to address `_recipient` with value `_amount` (in `Qa`). 
1. Continue on the traversing and to check for more deposits


@kaikawaliu

**Review by bb111189 — CHANGES_REQUESTED** [review]

File: docs/exchange-transaction-receipts.md (hunk 5 above)
Comment (bb111189) [inline docs/exchange-transaction-receipts.md]: For `Multiple Transitions with Balance Transfers`, I suggest we put it in our upcoming revamp of dev portal instead, so as to not confuse exchanges

**Review by bb111189 — APPROVED** [review]

**Review by kaikawaliu — APPROVED** [review]

File: docs/exchange-transaction-receipts.md (hunk 5 above)
Comment (kaikawaliu) [reply docs/exchange-transaction-receipts.md]: I suggest to check the `_tag` first since it's an easier search comparing to looking up the `_recipient`

**Pull request merged**